cellprofiler/modules/measureobjectneighbors.py

```diff
@@ -727,7 +729,7 @@ def run(self, workspace):
             count_image = Image(img, masking_objects=objects)
             image_set.add(self.count_image_name.value, count_image)
         else:
-            neighbor_cm_name = get_default_colormap()
+            neighbor_cm_name = "Blues"
```

Comment:
Why did we choose these defaults?

Reply:
Asking to make sure they are colorblind friendly.

Reply:
For future record - a single colour scale will always be fine, since it's the relative brightness that matters instead of colour hue.